README.md

```diff
@@ -90,10 +91,9 @@ If you’re participating in *Indie Support Weeks* and want to help promote some
 | [Diagram.Codes Studio](https://studio.diagram.codes) | [@hugozap](https://twitter.com/hugozap) | |
 | [Digital Photo Frame](https://itunes.apple.com/us/app/digital-photo-frame-pro-slideshow-creator/id1219786089?ls=1&mt=8) | [@manuelescrig](https://twitter.com/manuelescrig) | |
 | [Do I need it?](https://apps.apple.com/app/do-i-need-it/id1495111840) | [@maurogarcia_19](https://twitter.com/maurogarcia_19) | |
-| [DrawRun - Easily start running](https://apps.apple.com/us/app/drawrun-map-your-runs/id1442873786) | [@jordibruin](https://twitter.com/jordibruin) | SwiftLee |
+| [DrawRun - Easily start running](https://apps.apple.com/us/app/drawrun-map-your-runs/id1442873786) | [@jordibruin](https://twitter.com/jordibruin) | |
```

Comment:
Please revert these removals of others' markings 🙂

Reply:
It may just be a Git thing, or a mistake, so no worries, but I'd really appreciate it if you could fix these 🙂

Reply:
Opps, I think it's because of git  issue. I will make another PR

Reply:
Thanks 🙂